### PR TITLE
fmilib 2.0.1 (new formula)

### PR DESCRIPTION
**Pull request by jondo2010**
http://www.jmodelica.org/FMILibrary

### CODE REVIEW

File: Library/Formula/fmilib.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    args = %W[
+      -DFMILIB_INSTALL_PREFIX=#{prefix}
+    ]
+    if build.with? "debug"
+        args << "-DFMILIB_DEFAULT_BUILD_TYPE_RELEASE=false"
+    end
+    ENV.cxx11 if MacOS.version < :mavericks
+
+    mkdir "build"
+    cd "build" do
```
Comment (bfontaine) [inline Library/Formula/fmilib.rb]: You can use `mkdir "build" do`

File: Library/Formula/fmilib.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  depends_on "cmake" => :build
+
+  option "with-debug", "Build with debugging symbols"
+
+  def install
+    args = %W[
+      -DFMILIB_INSTALL_PREFIX=#{prefix}
+    ]
+    if build.with? "debug"
+        args << "-DFMILIB_DEFAULT_BUILD_TYPE_RELEASE=false"
```
Comment (bfontaine) [inline Library/Formula/fmilib.rb]: Please use 2-spaces indentation.

File: Library/Formula/fmilib.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+class Fmilib < Formula
+  desc "FMI Library enables integration of Functional Mock-up Units (FMUs)"
+  homepage "http://www.jmodelica.org/FMILibrary"
+  url "http://www.jmodelica.org/downloads/FMIL/FMILibrary-2.0.1-src.zip"
+  sha256 "2b82ad1bbb54399ef595755ca9e12e678e8d1ad0e227609d478d9934a298cc43"
+
+  depends_on "cmake" => :build
+
+  option "with-debug", "Build with debugging symbols"
```
Comment (bfontaine) [inline Library/Formula/fmilib.rb]: Please move this option above the `depends_on` (leave an empty line between them).

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by apjanke** [comment]
@BrewTestBot test this please

File: Library/Formula/fmilib.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  url "http://www.jmodelica.org/downloads/FMIL/FMILibrary-2.0.1-src.zip"
+  sha256 "2b82ad1bbb54399ef595755ca9e12e678e8d1ad0e227609d478d9934a298cc43"
+  head "https://svn.jmodelica.org/FMILibrary/trunk"
+
+  option "with-debug", "Build with debugging symbols"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = %W[
```
Comment (dunn) [inline Library/Formula/fmilib.rb]: The `std_cmake_args` includes a prefix argument, so this shouldn't be necessary.  You should be able to do something like
```ruby
args = std_cmake_args
args << "-DFMILIB_DEFAULT_BUILD_TYPE_RELEASE=false" if build.with? "debug"

**Comment by MikeMcQuaid** [comment]
Passing on this as-is, please reopen on https://github.com/homebrew/homebrew-core if desired.

**Pull request closed without merging**